test/turbopack-tests-manifest.json

```diff
@@ -4044,7 +4044,9 @@
       "app dir - rsc basics should track client components in dynamic imports",
       "app dir - rsc basics should use canary react for app"
     ],
-    "failed": [],
+    "failed": [
+      "app dir - rsc basics should be able to call legacy react-dom/server APIs in client components"
```

Comment:
let's make it work for turbopack? you'd need to update the next_import_map I think

Reply:
Looks like turbopack already supported that case, I changed a bit on alias now support it as well. Updated tests